includes/user-functions.php

```diff
@@ -531,25 +531,33 @@ function give_get_donor_address( $donor_id = null, $args = array() ) {
  *
  * Sends the new user notification email when a user registers within the donation form
  *
- * @param int   $user_id   User ID.
- * @param array $user_data An Array of User Data.
+ * @param int   $donation_id
+ * @param array $donation_data
  *
  * @access public
  * @since  1.0
  *
  * @return void
  */
-function give_new_user_notification( $user_id = 0, $user_data = array() ) {
+function give_new_user_notification( $donation_id = 0, $donation_data = array() ) {
```

Comment:
`$donation_data` contains:
```
Array
(
	[id] => 110
	[title] => 
	[email] => siddharth.thevaril@gmail.com
	[first_name] => Siddharth
	[last_name] => Thevaril
	[address] => 
	[donor_id] => 44
)
```

Reply:
`$user_data` and `$donation_data['user_info']` don't return the exact data and they also return data values with different keys.

CC: @ravinderk

Reply:
@Sidsector9 fill missing information for backward compatibility except `password`